k8-util/helm/fluvio-app/templates/fluvio-connector-config.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluvio-config-map
+data:
+  fluvioClientConfig: |
+    version = "2.0"
+    current_profile = "k8-config-map-deployment"
+
+    [profile.k8-config-map-deployment]
+    cluster = "k8-config-map-deployment"
+
+    [cluster.k8-config-map-deployment]
+    endpoint = "fluvio-sc-public:9003"
```

Comment:
is this test for real produce deployment config?.  can't hard code fluvio address/port number

Reply:
Ah yes. This works in dev but we will need to figure out this for prod. @nacardin thoughts?

Reply:
After a discussion after standup, we concluded we'd look at `{{ .Values.tls }}` and use `9005` if tls was enabled.